processor/metricstransformprocessor/operation_aggregate_labels.go

```diff
@@ -313,13 +317,15 @@ func mergeExponentialHistogramDataPoints(dpsMap map[string]pmetric.ExponentialHi
 			if dp.HasMax() && dp.Max() < dps.At(i).Max() {
 				dp.SetMax(dps.At(i).Max())
 			}
-			for b, bc := range dps.At(i).Negative().MBucketCounts() {
-				dps.At(i).Negative().MBucketCounts()[b] = dps.At(i).Negative().MBucketCounts()[b] + bc
+			for b := 0; b < dps.At(i).Negative().BucketCounts().Len(); b++ {
+				negatives[b] += dps.At(i).Negative().BucketCounts().At(b)
```

Comment:
`negatives` is a copy of `dp.Negative().BucketCounts().AsRaw()`. so `dp.Negative().BucketCounts().AsRaw()` should no be used here

Reply:
Why do you double them?

Reply:
Shouldn't the logic be sum of all counts in all "dps"?

Reply:
Oh nvm. I misread this, sorry